### PR TITLE
Fix various bugs prior to 0.8 release

### LINKED ISSUE #445: Max build heights not working on some maps
Likely due to region order conflicts, see `RegionApplicationParser`. This has occurred after the recent refactor.

### PR DESCRIPTION
**Pull request by Electroid**
Closes #445 and #446

### CODE REVIEW

File: core/src/main/java/tc/oc/pgm/PGMConfig.java [hunk 1]
```diff
@@ -40,6 +40,7 @@
 
   // database-uri
   private final String databaseUri;
+  private final int databaseParallelism;
```
Comment (rafibaum) [inline core/src/main/java/tc/oc/pgm/PGMConfig.java:43]: Odd naming. Consider `databaseConnections` or `pooledConnections`?

File: core/src/main/java/tc/oc/pgm/PGMPlugin.java [hunk 2]
```diff
@@ -242,18 +250,9 @@ public void onDisable() {
     if (matchTabManager != null) matchTabManager.disable();
     if (matchManager != null) matchManager.getMatches().forEachRemaining(Match::unload);
     if (vanishManager != null) vanishManager.disable();
-
-    // Sometimes match folders need to be cleaned up due to de-syncs
-    for (File dir : getServer().getWorldContainer().listFiles()) {
-      if (dir.isDirectory() && dir.getName().startsWith("match")) {
-        FileUtils.delete(dir);
-      }
-    }
```
Comment (rafibaum) [inline core/src/main/java/tc/oc/pgm/PGMPlugin.java:251]: Have the de-syncs been fixed?

File: core/src/main/java/tc/oc/pgm/api/Config.java [hunk 3]
```diff
@@ -28,6 +28,13 @@
   @Nullable
   String getDatabaseUri();
 
+  /**
+   * Gets the maximum number of concurrent database connections.
+   *
+   * @return Number of parallel connections.
+   */
+  int getDatabaseParallelism();
```
Comment (rafibaum) [inline core/src/main/java/tc/oc/pgm/api/Config.java:36]: Same comment as earlier about naming.

**Comment by Electroid** [comment]
Yea, we should definitely investigate ProGuard. I still think in this case our own utility might have been better, ORMs are huge by nature.

**Pull request merged**